sechub-doc/src/docs/asciidoc/sechub-getting-started/run-all-steps.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Get sechub's root folder
+currentDir=$(pwd)
+baseDir="$currentDir"
+regex=sechub$
+
+while ! [[ "$baseDir" =~ ${regex} ]]
+do
+baseDir="$(dirname $baseDir)"
+done
+
+
+# Copy all scripts from the current folder
+cp ./* "$baseDir"
+rm "$baseDir/run-all-steps.sh"
+cd $baseDir
+
+sechub_pid=0
+pds_pid=0
+
+stopServers(){
+    kill "$sechub_pid"
+    kill "$pds_pid"
+    exit 0
+}
+
+# Start SecHub
+$(sh ./start-sechub.sh) &
+sechub_pid=$!
+
+sleep 15
+
+# Start PDS+GoSec
+$(sh ./start-pds-gosec.sh) &
+pds_pid=$!
+
+sleep 15
+
+# Configure project and variables
+# sh ./setup-project.sh
+
+trap 'stopServers' INT
```

Comment:
This approach is okay, but I figured out that it is possible to use a simple stop script. Have a look at: sechub-pds-solutions/tern/01-stop-single-docker-compose.sh. My prefered way would be to just add stop scripts to the solution and pds-gosec solution.